compose/internal/composebinary/composebinary.go

```diff
@@ -104,7 +104,7 @@ func newCommand(command []string, filePaths []string) composeCommand {
 }
 
 func newUpCommand(filePaths []string) composeCommand {
-	return newCommand([]string{"up", "-d"}, filePaths)
+	return newCommand([]string{"up", "-d", "--force-recreate"}, filePaths)
```

Comment:
will it be better if we can use a parameter here to control if we need to add `--force-recreate` arg or not?

Reply:
hmmmmm, yeh, I think in our stack deployment scenario it should always be a force recreate